packages/babel-plugin/src/css-prop/__tests__/behaviour.test.tsx

```diff
@@ -23,7 +19,7 @@ describe('css prop behaviour', () => {
       <div css={{}} />
     `);
 
-    expect(actual).toInclude('<div className="cc-hash-test"/>');
+    expect(actual).toInclude('<div className=""/>');
```

Comment:
Should we remove blank classname? It will endup in the bundle, as well as in DOM.

Reply:
yeah sounds good will do tmr